cmd/metal-api/internal/metal/image.go

```diff
@@ -3,15 +3,63 @@ package metal
 import (
 	"fmt"
 	"strings"
+	"time"
 )
 
 // An Image describes an image which could be used for provisioning.
 type Image struct {
 	Base
 	URL      string                    `rethinkdb:"url" json:"url"`
 	Features map[ImageFeatureType]bool `rethinkdb:"features" json:"features"`
+	// OS is parsed from id and is the first part, specifies operating system derivate, internal usage only
+	OS string `rethinkdb:"os" json:"os"`
```

Comment:
Unrelated but I am confused that we have `json` annotations here again... I remember that I removed all of them at some point in time. Not sure why and when they came back.